packages/lib-classifier/src/plugins/tasks/SubjectGroupComparisonTask/models/SubjectGroupComparisonAnnotation.js

```diff
@@ -1,15 +1,20 @@
 import { types } from 'mobx-state-tree'
 import Annotation from '../../models/Annotation'
 
-const SubjectGroup = types.model('SubjectGroup', {
-  value: types.optional(types.array(types.number), [])
+const SubjectIndexPair = types.model('SubjectIndexPair', {
+  index: types.number,
+  subject: types.string,
+})
+
+const SubjectGroupComparison = types.model('SubjectGroupComparison', {
+  value: types.optional(types.array(SubjectIndexPair), [])
```

Comment:
I'm pretty sure `types.array` defaults to an empty array, so `types.optional` isn't needed here.

Reply:
👍